src/sgame/sg_admin.cpp

```diff
@@ -1184,7 +1184,7 @@ static bool admin_match( void *admin, const void *match )
 	}
 
 	G_SanitiseString( ( ( g_admin_admin_t * ) admin )->name, n1, sizeof( n1 ) );
-	return strstr( n1, n2 ) ? true : false;
+	return !!strstr( n1, n2 );
```

Comment:
The fact is, in original code:

```c
return strstr( n1, n2 ) ? true : false;
```

`strstr( n1, n2 )` is not a boolean, so it's an issue for the `?` test even if it's not an issue for the `return`

I guess the best way to fix it would be to do:

```c
return strstr( n1, n2 ) != NULL;
```

What do you think about it?